File: Codeforces/Boy_Girl.cpp
#include<iostream>
#include<string.h>
using namespace std;
int main()
{
    int temp=0,flag=0;
    string str;
    cin>>str;
    for(int i=0;i<str.length();i++)
    {
        for(int j=0;j<i;j++)
        {
            if(str.at(i)==str.at(j))
            {
                temp++;
                break;
            }
        }

    }
    //cout<<str.length()-temp;
    if((str.length()-temp)%2==0)
        cout<<"CHAT WITH HER!";
    else
        cout<<"IGNORE HIM!";
}

File: Codeforces/BoyGirl.java
import java.util.Scanner;
public class BoyGirl
{
	public static void main(String asd[])
	{
		Scanner sc=new Scanner(System.in);
		String s=sc.nextLine();
		int count=0;
		
		char temp[] = s.toCharArray();
		for(int i=0;i<s.length();i++)
			{
				char k=s.charAt(i);
				for(int j=0;j<i;j++)
				{
					if(i!=j)
					{
						if(k==temp[j])
						{
							count++;
						}
					}
					
				}
			}

			System.out.println(count);
		/*int l=s.length();
		int b=l-count+1;
		if(b%2==0)
		{
			System.out.println("CHAT WITH HER!");
		}
		else
		System.out.println("IGNORE HIM!");*/
		sc.close();
	}
}